Getting-Started/Setup/Server-Setup/Load-Balancing/file-system-replication.md

```diff
@@ -0,0 +1,72 @@
+---
+versionFrom: 8.0.0
+---
+
+## Non-synchronised/replicated file system
+
+If the file system on your servers isn't performing any file replication then no _Umbraco_ configuration file changes are necessary.
```

Comment:
I'm not sure people would know, or whether this describes the scenario well enough?

eg if Media is configured to be in blob storage, and changes aren't made to templates on teh production environment, then no form of file replication is required, and no specific umbraco configuration changes are required either?